connectors/src/connectors/webcrawler/index.ts

```diff
@@ -443,10 +443,27 @@ export async function setWebcrawlerConfiguration(
   if (stopRes.isErr()) {
     return new Err(stopRes.error);
   }
-  const startRes = await launchCrawlWebsiteWorkflow(connector.id);
+
+  const queueName = webcrawlerConfig.lastCrawledAt ? 0 : 1;
```

Comment:
queueName is definitely ill named here. Agree with @lasryaric that we want a clean type at the client interface